  fun had_error(task': GTaskI): I32 =>
    @g_task_had_error(task'.getptr())
